lib/bone.js

```diff
@@ -1339,6 +1388,7 @@ class Bone {
       if (attributes.hasOwnProperty(snakeCase(name))) {
         timestamps[name] = snakeCase(name);
       }
+      if (attributes.hasOwnProperty(LEGACY_TIMESTAMP_MAP[name])) timestamps[name] = LEGACY_TIMESTAMP_MAP[name];
```

Comment:
`Bone.init` isn't an async function , cannot  call `querySchemaInfo` in it, so take a `LEGACY_TIMESTAMP_MAP ` patch here too.